src/operations/mod.rs

```diff
@@ -41,18 +44,22 @@ pub enum OperationType {
     Append = 9,
     Prepend = 10,
     Touch = 11,
+    BitRead = 12,
+    BitWrite = 13,
+    Delete = 14,
+    HllRead = 15,
+    HllWrite = 16,
```

Comment:
There is only one match that takes in the OperationType. Its the buffer.rs line 402.
But there is no placeholder for HLL because it used a catch all in the end anyways.
As far as i remember, the HLL enum types should never be used yet and no changes have been made for HLL yet.